src/traces/scattergl/index.js

```diff
@@ -295,13 +295,27 @@ function sceneUpdate(gd, subplot) {
 
         // remove scene resources
         scene.destroy = function destroy() {
-            if(scene.fill2d) scene.fill2d.destroy();
-            if(scene.scatter2d) scene.scatter2d.destroy();
-            if(scene.error2d) scene.error2d.destroy();
-            if(scene.line2d) scene.line2d.destroy();
-            if(scene.select2d) scene.select2d.destroy();
+            if((scene.fill2d) &&
```

Comment:
Non-blocking comment: We (me and @alexcjohnson ) would usually write these lines as

```js
if(scene.fill2d && scene.fill2d.destroy) {
    scene.fill2d.destroy();
}